### PR TITLE
appveyor: add license notice about VC++ distributale code

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

**Comment by kenhys** [comment]
just waiting appveyor result.

**Comment by kenhys** [comment]
@komainu8 , @kou 
Is it enough to note about redistributable code? (Does it need more something?)

File: packages/windows/vcruntime/vs2013/redist.txt [hunk 1]
```diff
@@ -1,9 +1,14 @@
-This package contains Visual C++ Runtime files:
+Subject to Microsoft Licensing Product Use Rights [1], This package
+contains Visual C++ Runtime files as redistributable code:
+
+[1] http://download.microsoft.com/download/7/8/6/786cbfca-6826-4c58-9f4c-ee2067e619f5%2FPUR0604.pdf
```
Comment (komainu8) [inline packages/windows/vcruntime/vs2013/redist.txt]: This PDF is Japanese.
It is OK if we don't prepare a link to the English version?

Comment (kenhys) [reply packages/windows/vcruntime/vs2013/redist.txt]: Ah, it is better to prepare English version.

Comment (kenhys) [reply packages/windows/vcruntime/vs2013/redist.txt]: I can't find english version PUR which is corresponding to Japanese PUR ...

Comment (cosmo0920) [reply packages/windows/vcruntime/vs2013/redist.txt]: For VS 2013, we may use this document:
https://visualstudio.microsoft.com/license-terms/dn877550/

File: packages/windows/vcruntime/vs2015/redist.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Subject to Microsoft Licensing Product Use Rights [1], This package
+contains Visual C++ Runtime files as redistributable code:
+
+[1] http://download.microsoft.com/download/7/8/6/786cbfca-6826-4c58-9f4c-ee2067e619f5%2FPUR0604.pdf
```
Comment (cosmo0920) [inline packages/windows/vcruntime/vs2015/redist.txt]: For VC++ 2015 redistributables, we can use this documentation:
https://visualstudio.microsoft.com/license-terms/mt171552/

File: packages/windows/vcruntime/vs2017/redist.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Subject to Microsoft Licensing Product Use Rights [1], This package
+contains Visual C++ Runtime files as redistributable code:
+
+[1] http://download.microsoft.com/download/7/8/6/786cbfca-6826-4c58-9f4c-ee2067e619f5%2FPUR0604.pdf
```
Comment (cosmo0920) [inline packages/windows/vcruntime/vs2017/redist.txt]: For VC++ 2017 redistributables, we can use this documentation:
https://visualstudio.microsoft.com/license-terms/mlt552233/

**Comment by cosmo0920** [comment]
@kenhys I've added suggested changes comments. Could you kindly take a look?

**Comment by kenhys** [comment]
Thanks! @cosmo0920 !

**Comment by kenhys** [comment]
@komainu8 I've fixed link to them. (except vs2017, japanese edition is not published)

File: packages/windows/vcruntime/vs2015/readme.txt [hunk 4]
```diff
@@ -0,0 +1,15 @@
+Subject to Microsoft Visual Studio Community 2015 EULA [1], This package
+contains Visual C++ Runtime files as redistributable code:
+
+[1] https://visualstudio.microsoft.com/license-terms/microsoft-visual-studio-community-2015/
```
Comment (komainu8) [inline packages/windows/vcruntime/vs2015/readme.txt]: I think this link is broken.
I tried to display this page in the browser(Firefox). However, I could not display this page.

File: packages/windows/vcruntime/vs2017/readme.txt [hunk 5]
```diff
@@ -0,0 +1,17 @@
+Subject to Microsoft Visual Studio Community 2017 EULA [1], This package
+ contains Visual C++ Runtime files as redistributable code:
+
+[1] https://visualstudio.microsoft.com/license-terms/mlt553321/
```
Comment (komainu8) [inline packages/windows/vcruntime/vs2017/readme.txt]: This link connects to the Japanese page. At least, in my environment.

File: packages/windows/vcruntime/vs2015/readme.txt (hunk 4 above)
Comment (kenhys) [reply packages/windows/vcruntime/vs2015/readme.txt]: it seems that maybe  bottom language setting is set to "Japanese" for mirosoft.com.

File: packages/windows/vcruntime/vs2017/readme.txt (hunk 5 above)
Comment (kenhys) [reply packages/windows/vcruntime/vs2017/readme.txt]: It also depends on language navigation setting on bottom of website

File: packages/windows/vcruntime/vs2015/readme.txt (hunk 4 above)
Comment (komainu8) [reply packages/windows/vcruntime/vs2015/readme.txt]: Oh.
Thank you for your information.
I could display this page when I changed the language setting to "English".

File: packages/windows/vcruntime/vs2017/readme.txt (hunk 5 above)
Comment (komainu8) [reply packages/windows/vcruntime/vs2017/readme.txt]: Oh.
Thank you for your information.
I could display the English page when I changed the language setting to "English".

**Pull request merged**